.github/workflows/backport.yml

```
@@ -1,25 +1,30 @@
 name: backport.yml
 on:
-    pull_request:
-      types: [ closed, labeled ]
-  # pull_request_target:
-  #   types: [ labeled, closed ]
+    #pull_request:
+    #  types: [ closed, labeled ]
+    pull_request_target:
+      types: [ labeled, closed ]
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)